src/constants/remoteConfigConstants.js

```diff
@@ -40,6 +40,12 @@ export const REMOTE_CONFIG = {
   FEATURE_ONBOARDING: 'feature_onboarding',
   ENS_MIGRATOR_ENABLED: 'ens_migrator_enabled',
   KNOWLEDGE_BASE_URL: 'knowlegebase_url',
+  ANDROID_FEATURE_STORIES_HEIGHT: 'feature_stories_android_height',
```

Comment:
Could we keep the naming consistent please with the feature flag name?
e.g
```suggestion
  FEATURE_STORIES_ANDROID_HEIGHT: 'feature_stories_android_height',
```

Reply:
Done.